server/serverAPI/controller/InventoryController.ts

```diff
@@ -12,11 +13,8 @@ import BaseUserController from "./BaseUserController";
  * provided to the user.
  */
 export default class InventoryController extends BaseUserController {
-    private ingredientAPI: IIngredientAPI;
-
-    constructor(database: IDatabase<IUser>, ingredientAPI: IIngredientAPI) {
+    constructor(database: IDatabase<IUser>) {
         super(database);
-        this.ingredientAPI = ingredientAPI;
```

Comment:
Why is this being removed? If you don't need it anymore, you can remove the ingredientAPI import

Reply:
We are cleaning up :triumph: